packages/core/src/Select/Select.stories.tsx

```diff
@@ -92,6 +92,90 @@ export const Variants: StoryObj<HvSelectProps<{}, false>> = {
   },
 };
 
+export const Variants: StoryObj<HvSelectProps<{}, false>> = {
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "By default the select button uses the `secondarySubtle` variant. However, if needed, any `HvButton` `variant` can be used.",
```

Comment:
I think it's overkill to add `size` and `variant` to every component extending `HvButton`. These "variants" will rarely be used, and IMO don't add much value to the docs since we already have the args table. We could add a default value to `size` and `variant` in `Main.args` so that these props are "bumped up"

What do you think?